### ISSUE TITLE
Func .prev() does not return

### ISSUE THREAD

**Issue by marcoprado17** [harrisiirak/cron-parser#131]
Hi Guys

I am using the package and it help me a lot but I found a problem:

In the script bellow, the function .prev does not return

```
const cronParser = require('cron-parser');

let interval = cronParser.parseExpression("0 9 1 1 *", {tz: "America/Sao_Paulo"});

console.log(interval.next());
console.log(interval.prev());
```

node version: v8.11.3
cron-parser version: 2.5.0

Thanks

**Comment by jsemander** [comment]
Something about the timezone puts it into an infinite loop.

When changing to another time zone, it seems to work fine.

```
const parser = require('./lib/parser');
const interval = parser.parseExpression('0 9 1 1 *', {
    tz: 'America/New_York',
});
console.log(interval.next());
console.log(interval.prev());
```

I can check this out.

**Comment by mehuldudi** [comment]
I'm also facing a similar issue. Did you find anything?

**Comment by marcoprado17** [comment]
@mehuldudi 

I solve the deadlock problem creating one instance of interval to get the next cron date and another instance to get the prev cron date:

`
const cronParser = require('cron-parser');

let nextInterval = cronParser.parseExpression("0 9 1 1 *", {tz: "America/Sao_Paulo"}).next();
let prevInterval = cronParser.parseExpression("0 9 1 1 *", {tz: "America/Sao_Paulo"}).prev();

console.log(nextInterval);
console.log(prevInterval);
`

**Comment by yoavapi** [comment]
thanks @marcoprado17  you solved my issue.

**Comment by harrisiirak** [comment]
Released fix in 2.7.0
